### PR TITLE
(rc) document inventories_checks_configuration_enabled being enabled …

### PR DESCRIPTION
**Pull request by arbll**
…by default

### What does this PR do?

`inventories_checks_configuration_enabled` is now set to `true` by default since https://github.com/DataDog/datadog-agent/pull/19485.

Document it in the release note and config template.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by ogaca-dd — APPROVED** [review]

File: releasenotes/notes/send-integration-config-default-05a00f3e99612e6b.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The agent now sends its integration / checks configuration to the backend by default. They will
+    soon be available in the Datadog UI. The configurations are scrubbed of any sensitive information.
```
Comment (hush-hush) [inline releasenotes/notes/send-integration-config-default-05a00f3e99612e6b.yaml:12]:
```suggestion
    The agent now sends its integration / checks configuration to the backend by default. They will
    soon be available in the Datadog UI. The configurations are scrubbed of any sensitive information.
    Set `inventories_checks_configuration_enabled` to false to disable this feature.
```

**Review by hush-hush — CHANGES_REQUESTED** [review]

File: pkg/config/config_template.yaml [hunk 2]
```diff
@@ -495,6 +495,14 @@ api_key:
 #
 # inventories_configuration_enabled: true
 
+## @param inventories_checks_configuration_enabled - boolean - optional - default: true
+## @env DD_INVENTORIES_CHECKS_CONFIGURATION_ENABLED - boolean - optional - default: true
+## By default the Agent sends its integrations / checks configuration to Datadog to be displayed in the UI.
```
Comment (alai97) [inline pkg/config/config_template.yaml:500]:
```suggestion
## By default the Agent sends its integrations and checks configuration to Datadog to be displayed in the UI.
```

File: releasenotes/notes/send-integration-config-default-05a00f3e99612e6b.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The agent now sends its integration / checks configuration to the backend by default. They will
```
Comment (alai97) [inline releasenotes/notes/send-integration-config-default-05a00f3e99612e6b.yaml:11]:
```suggestion
    The Agent now sends its integration and checks configuration to the backend by default. They will
```

File: releasenotes/notes/send-integration-config-default-05a00f3e99612e6b.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    soon be available in the Datadog UI. The configurations are scrubbed of any sensitive information.
```
Comment (alai97) [inline releasenotes/notes/send-integration-config-default-05a00f3e99612e6b.yaml:12]:
```suggestion
   be available in the Datadog UI. The configurations are scrubbed of any sensitive information.
```

**Review by alai97 — APPROVED** [review]
Suggestions to the release note!

**Review by sgnn7 — APPROVED** [review]
+1 on Austin's comments but overall LGTM

File: pkg/config/config_template.yaml [hunk 5]
```diff
@@ -495,6 +495,14 @@ api_key:
 #
 # inventories_configuration_enabled: true
 
+## @param inventories_checks_configuration_enabled - boolean - optional - default: true
+## @env DD_INVENTORIES_CHECKS_CONFIGURATION_ENABLED - boolean - optional - default: true
+## By default the Agent sends its integrations and checks configuration to Datadog to be displayed in the UI.
```
Comment (hush-hush) [inline pkg/config/config_template.yaml:500]: I think we use `integration` for customers
```suggestion
## By default the Agent sends the configuration for integrations being run to Datadog to be displayed in the UI.
```

**Review by hush-hush — APPROVED** [review]
Small nit-pick, feel free to merge after :+1:

**Pull request closed without merging**